### PR TITLE
Chore/upgrade to Circle CI 2 using transitional machine

### PR DESCRIPTION
**Pull request by santiagonoguez**
This uses a docker image I created from the transitional image CCI2 tool gave me. Node version configuration, and OS packages are included there, also chrome upgrade.

Latest build: https://circleci.com/workflow-run/201094e0-a6ed-414c-af6c-f874776ffe32

A build  based on our e2e images still failed. Branch here:
https://github.com/Rise-Vision/widget-settings-ui-components/tree/chore/update_to_circle_ci_2

Also had to do some tweaking to some font tests that were failing.

### CODE REVIEW

File: test/e2e/angular/font-setting-test-ng.js [hunk 1]
```diff
@@ -275,14 +275,14 @@
       });
 
       describe("Google Font", function() {
-        it("should update font family for preview text if family name is one word", function() {
+        xit("should update font family for preview text if family name is one word", function() {
```
Comment (stulees) [inline test/e2e/angular/font-setting-test-ng.js]: @santiagonoguez Curious, why did these two tests need turning off?

Comment (santiagonoguez) [reply test/e2e/angular/font-setting-test-ng.js]: @stulees The problem was that the selector id used in the tests no longer exist in the component; 

I changed the selection by font name instead so it depends less on ids that we do not control.

Also, I changed the fonts used for these tests as previous one require scrolling on the select box before clicking. If we want to use the original fonts for these tests, that behavior should be coded into them also, but that may be another PR.

Comment (stulees) [reply test/e2e/angular/font-setting-test-ng.js]: @santiagonoguez The problem with this I see now is that these tests are specifically for Google fonts, and now you're targeting Arial. 

In the past, if we needed to find the selector id (due to a Google font order change), we'd test the component manually in browser and inspect the HTML to find the appropriate selector ID. 

I'd prefer we do this but don't want you to spend more time with this, so perhaps we go back to turning the tests off and revisit. Cheers

Comment (santiagonoguez) [reply test/e2e/angular/font-setting-test-ng.js]: @stulees I added code to scroll to the bottom of the font layer, and select Google fonts there. Please review again.

**Pull request merged**